.gitignore

```diff
@@ -1 +1,3 @@
 *.swp
+/maas-*-alert-rules
+/*-alert-rules.yml
```

Comment:
minor, but mind fixing the line ending here?

Reply:
done